content/v2.0/write-data/use-telegraf/use-http-plugin.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Use the HTTP input plugin with Citi Bike data
+description:
+menu:
+  v2_0:
+    parent: Use Telegraf
+weight: 202
+---
+
+This example walks through using the Telegraf HTTP input plugin to collect live metrics on Citi Bike stations in New York City. Live station data is available in JSON format from [NYC OpenData](https://data.cityofnewyork.us/NYC-BigApps/Citi-Bike-Live-Station-Feed-JSON-/p94q-8hxh).
+
+For the following example to work, configure [`influxdb` output plugin](telegraf/v1.13/plugins/plugin-list/#influxdb). This plugin is what allows Telegraf to write the metrics to your InfluxDB.
```

Comment:
Do we want to send them here?
https://v2.docs.influxdata.com/v2.0/write-data/use-telegraf/manual-config/#example-influxdb-v2-configuration